### ISSUE TITLE
COAM\ Earned credits - Internal user can view track status of issuance

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#102]
#### Description:

CAS staff are able to view the status of the earned credits issuance request.

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #63 

#### Notes:

-

**Comment by vesselak** [comment]
Refinement Notes 06/11:
Updating the links to Figma @zoeyli-46 please review the notification text in the ticket 
@nataliepiguliak please add dev checklist and review estimate before starting the work. thank you

**Comment by zoeyli-46** [comment]
The notification text and status text and wireframe links have been updated in this ticket

**Comment by fviduya** [comment]
Hi @nataliepiguliak, I can't seem to test this in DEV. When I go to Compliance Summaries, I get this page: 

![Image](https://github.com/user-attachments/assets/63f895ad-fef0-45fe-bf76-aaae70c1e92d)

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/EVZJgyG63HBMgwn_9_luqzkBfZP-eBfWsWkqFuz9fhzI3g?e=jceL2N).

I can verify that, as a CAS staff member, given that an operator has requested an EC issuance:
- [x] The record of issuance request for the operation is visible in the Compliance Summaries grid
~- [ ] The action name is `Review Credits Issuance Request` - **Current action name is: `Request Issuance of Credits`**~ - **#217**
- [x] When they click the link in the action column and click `Track Status of Issuance`, they can view the status of the issuance request 
- [x] The following is visible in the Track Status of Issuance page:
   - [x] Header: Track Status of Issuance
   - [x] Subheading: Earned Credits
   - [x] Notification message
   - [x] Earned Credits quantity
   - [x] Status of Issuance
   - [x] BCCR Trading Name
   - [x] BCCR Holding Account ID
   - [x] Director's Comments
- [x] When the request is still outstanding: - **The page does not show the notification message and status of issuance is Approved** - **EDIT: This is hardcoded for now, will display Approved by default**
   - [x] Notification message is visible in the Review by Director's page: Once the issuance request is approved, the earned credits will be issued to the holding account as identified above in B.C. Carbon Registry.
   - [x] Status of issuance: Issuance requested, awaiting approval
- [x] When the request has been approved:
   - [x] Notification message: The issuance request is approved. The earned credits have been issued to the holding account as identified below in B.C. Carbon Registry (BCCR) successfully.
   - [x] Status of issuance: Approved, credits issued in BCCR

Hi @nataliepiguliak, the notification message does not show in Review by Director page when request is still outstanding. Is this hardcoded to be approved for now? cc: @vesselak

**Comment by nataliepiguliak** [comment]
@fviduya It is hardcoded for now, pending API changes

**Comment by fviduya** [comment]
Thanks for confirming! Moving this to Pending PO for review. The action name will be renamed in #217.

**Comment by vesselak** [comment]
Reviewed testing documentation and accepting ticket with the changes that will be completed in #217 and the API integration as part of #166
